### ISSUE TITLE
core.js fireEvent('submit') IE issue

### ISSUE THREAD

**Issue by dentrite** [joomla/joomla-cms#5190]
https://github.com/joomla/joomla-cms/blob/staging/media/system/js/core-uncompressed.js#L30

IE10 debugger prompts "invalid parameter"(translated from Chinese) when trying to submit form through the core.js script. After searching the web, I found a solution:
```
 fireEvent('onsubmit')    // instead of fireEvent('submit') 
```
The problem disappeared after the change.

**Comment by brianteeman** [comment]
Sorry that it has taken so long for you to get a reply on this issue. Is it still a problem in the current release of joomla or has it been resolved elsewhere?<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/5190">issues.joomla.org/joomla-cms/5190</a>.</sub>

**Comment by Fedik** [comment]
this issue already fixed in #6587

**Comment by zero-24** [comment]
Closing than. Thanks
